src/imitation/algorithms/preference_comparisons.py

```diff
@@ -317,6 +317,144 @@ def _get_trajectories(
     return trajectories
 
 
+class PreferencePredictor(nn.Module):
```

Comment:
Might call this `PreferenceModel` instead of preference predictor since it is effectively a likelihood model.

Reply:
`PreferenceModel` sounds better!